### PR TITLE
Apply @kidayasuo's proposal

### PR DESCRIPTION
**Pull request by kojiishi**

### CODE REVIEW

**Comment by kojiishi** [comment]
@kidayasuo

File: auto-spacing.txt [hunk 1]
```diff
@@ -592,17 +592,20 @@
 2FF0..2FFB     ; O  # W   IDEOGRAPHIC DESCRIPTION CHARACTER LEFT TO RIGHT..IDEOGRAPHIC DESCRIPTION CHARACTER OVERLAID
 3000           ; O  # F   IDEOGRAPHIC SPACE
 3001..3004     ; O  # W   IDEOGRAPHIC COMMA..JAPANESE INDUSTRIAL STANDARD SYMBOL
-3005..3007     ; W  # W   IDEOGRAPHIC ITERATION MARK..IDEOGRAPHIC NUMBER ZERO
-3008..3020     ; O  # W   LEFT ANGLE BRACKET..POSTAL MARK FACE
-3021..302D     ; W  # W   HANGZHOU NUMERAL ONE..IDEOGRAPHIC ENTERING TONE MARK
-302E..302F     ; N  # W   HANGUL SINGLE DOT TONE MARK..HANGUL DOUBLE DOT TONE MARK
+3005           ; N  # W   IDEOGRAPHIC ITERATION MARK
+3006           ; W  # W   IDEOGRAPHIC CLOSING MARK
+3007..3029     ; O  # W   IDEOGRAPHIC NUMBER ZERO..HANGZHOU NUMERAL NINE
```
Comment (kojiishi) [inline auto-spacing.txt:597]: [U+3007](https://codepoints.net/U+3007): Shouldn't this stay `W`, no?

File: auto-spacing.txt [hunk 2]
```diff
@@ -592,17 +592,20 @@
...
 3031..3035     ; W  # W   VERTICAL KANA REPEAT MARK..VERTICAL KANA REPEAT MARK LOWER HALF
-3036..3037     ; O  # W   CIRCLED POSTAL MARK..IDEOGRAPHIC TELEGRAPH LINE FEED SEPARATOR SYMBOL
-3038..303C     ; W  # W   HANGZHOU NUMERAL TEN..MASU MARK
+3036..303A     ; O  # W   CIRCLED POSTAL MARK..HANGZHOU NUMERAL THIRTY
+303B           ; N  # W   VERTICAL IDEOGRAPHIC ITERATION MARK
+303C           ; W  # W   MASU MARK
 303D..303E     ; O  # W   PART ALTERNATION MARK..IDEOGRAPHIC VARIATION INDICATOR
 3041..3096     ; W  # W   HIRAGANA LETTER SMALL A..HIRAGANA LETTER SMALL KE
-3099..309F     ; W  # W   COMBINING KATAKANA-HIRAGANA VOICED SOUND MARK..HIRAGANA DIGRAPH YORI
+3099..309A     ; N  # W   COMBINING KATAKANA-HIRAGANA VOICED SOUND MARK..COMBINING KATAKANA-HIRAGANA SEMI-VOICED SOUND MARK
```
Comment (kojiishi) [inline auto-spacing.txt:606]: This looks like a regression to me, though they are Non-spacing Marks. WDYT?

File: auto-spacing.txt [hunk 3]
```diff
@@ -592,17 +592,20 @@
...
+309B..309C     ; O  # W   KATAKANA-HIRAGANA VOICED SOUND MARK..KATAKANA-HIRAGANA SEMI-VOICED SOUND MARK
```
Comment (kojiishi) [inline auto-spacing.txt:607]: This one too.

File: auto-spacing.txt (hunk 1 above)
Comment (kidayasuo) [reply auto-spacing.txt:597]: Yes, we discussed this at JLReq TF. As U+3007 is used together with Han numerals, it should be W.
U+3005 should also be W.

File: auto-spacing.txt (hunk 2 above)
Comment (kidayasuo) [reply auto-spacing.txt:606]: How the property value works on nonspacing marks?

Comment (kojiishi) [reply auto-spacing.txt:606]: > How the property value works on nonspacing marks?

As commented in another PR, it is applied when the code point appears alone in a paragraph. So minor edge case, but still better to be `W`?

**Comment by kojiishi** [comment]
@kidayasuo Is this now ok to close?

**Comment by kidayasuo** [comment]
> @kidayasuo Is this now ok to close?

Have U+3005 and 3007 been fixed? They should be W.

**Comment by kojiishi** [comment]
> Have U+3005 and 3007 been fixed? They should be W.

Yes, they're W now.

**Comment by kojiishi** [comment]
Closing as per the discussions above and no further feedback.

**Pull request closed without merging**